### PR TITLE
IFATQCZZ-9 - Added TestMerchandiserAfterLogInPage

### PR DESCRIPTION
**Pull request by kvitomasya**
Made minor changes in "UserInfoPage" and "MerchandiserOrderingPage"

### CODE REVIEW

File: src/test/java/academy/softserve/edu/tests/TestMerchandiserAfterLogInPage.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+public class TestMerchandiserAfterLogInPage {
+
+    public static final String LOG_IN_PAGE =
+            "http://192.168.56.101:8080/oms5/login.htm";
+
+    private WebDriver driver;
+    private UserInfoPage userInfoPage;
+
+    @DataProvider
```
Comment (toleksyn) [inline src/test/java/academy/softserve/edu/tests/TestMerchandiserAfterLogInPage.java]: create a separate class for data providers - DataProviders.java, move it to utils. share this change with the team

Comment (kvitomasya) [reply src/test/java/academy/softserve/edu/tests/TestMerchandiserAfterLogInPage.java]: done

File: src/test/java/academy/softserve/edu/tests/TestMerchandiserAfterLogInPage.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+public class TestMerchandiserAfterLogInPage {
+
+    public static final String LOG_IN_PAGE =
+            "http://192.168.56.101:8080/oms5/login.htm";
+    public static final int TIMEOUT = 10;
+
+    protected WebDriver driver;
+
+    @DataProvider
```
Comment (toleksyn) [inline src/test/java/academy/softserve/edu/tests/TestMerchandiserAfterLogInPage.java]: move out to dataprovider

File: src/test/java/academy/softserve/edu/tests/merchandiser/MerchandiserAfterLogInPageTest.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
+package academy.softserve.edu.tests.merchandiser;
```
Comment (toleksyn) [inline src/test/java/academy/softserve/edu/tests/merchandiser/MerchandiserAfterLogInPageTest.java:1]: add log messages to all asserts

Comment (kvitomasya) [reply src/test/java/academy/softserve/edu/tests/merchandiser/MerchandiserAfterLogInPageTest.java:1]: done

**Pull request merged**